package.json

```diff
@@ -26,7 +26,8 @@
     "loopback-connector": "^2.2.0",
     "mongodb": "~2.0.35",
```

Comment:
IIUC, the issue has been fixed in mongodb@2.1.18, see https://github.com/mongodb/node-mongodb-native/commit/51eb0aeac736b35e7397f294ec38c4c2e1e1671d. Instead of adding `kerberos` as a dependency, we should change this line to `"mongodb": "^2.1.18"`.

@raymondfeng @superkhau do you happen to know why we were using `~` operator preventing upgrades to newer minor versions? Is there any reason against switching to `^` operator?